custom/ilsgateway/templates/ilsgateway/bootstrap3/datespan.html

```diff
@@ -0,0 +1,95 @@
+{% extends 'reports/filters/bootstrap3/base.html' %}
+{% load hq_shared_tags %}
+{% block filter_content %}
+<div id="0">
+    <div>
+        <select id="date_span_type"
+            style="width: 100%"
+            data-bind="options: select_params,
+                       optionsText: 'text',
+                       optionsValue: 'val',
+                       value: selectedType,
+                       select2: {}"
+            name="datespan_type"></select>
+    </div>
+    <br />
+    <!-- ko if: firstOptions().length > 1 -->
+    <div>
+        <select id="date_span_first"
+            style="width: 100%"
+            data-bind="options: firstOptions,
+                       optionsText: 'text',
+                       optionsValue: 'val',
+                       value: first_selected,
+                       select2: {}"
+            name="datespan_first"></select>
+    </div>
+    <!-- /ko -->
+    <br />
+    <div>
+        <select id="date_span_second"
+            style="width: 100%"
+            data-bind="options: secondOptions,
+                       optionsText: 'text',
+                       optionsValue: 'val',
+                       value: second_selected,
+                       select2: {}"
+            name="datespan_second"></select>
+    </div>
+</div>
+
+{% endblock %}
+{% block filter_js %}
+<script type="text/javascript">
+    $(function () {
+        ko.bindingHandlers.select2 = {
+            init: function(element, valueAccessor) {
+
+                 var options = ko.toJS(valueAccessor()) || {};
+                  setTimeout(function() {
+                      $(element).select2(options);
+                  }, 0);
+               //$(element).select2(valueAccessor());
```

Comment:
was this meant to be commented out? i think we should remove the line if it's never going to be used